### PR TITLE
Enh: Add Configure Settings button

### PR DESCRIPTION
**Pull request by Felli**
Was tired of having to go to the Modules List just to get to the configure settings, so now there is a button that will take you to the configure settings via the no updates page. :+1:

### CODE REVIEW

File: views/update/index_noupdate.php [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+?>
 <div class="panel panel-default">
-    <div class="panel-heading"><?php echo Yii::t('UpdaterModule.base', '<strong>Update</strong> HumHub'); ?></div>
+    <div class="panel-heading"><?= Yii::t('UpdaterModule.base', '<strong>Update</strong> HumHub'); ?></div>
     <div class="panel-body">
         
-        <?php echo Yii::t('UpdaterModule.base', 'There is no new HumHub update available!'); ?>
-        <br />
-        
+        <?= Yii::t('UpdaterModule.base', 'There is no new HumHub update available!'); ?>
```
Comment (Felli) [inline views/update/index_noupdate.php]: Maybe for a grammar fix `is` should be changed to `are`. :thinking:

**Comment by ArchBlood** [comment]
Just noticed that the configuration button was removed from the index? Of course @greenmeteor reapplied this on our end, just wondering if this was intended @luke-
Screenshot_1

**Comment by luke-** [comment]
What do you mean with removed?

https://github.com/humhub/humhub-modules-updater/blob/master/views/update/index_noupdate.php#L21

**Comment by ArchBlood** [comment]
> What do you mean with removed?
> 
> https://github.com/humhub/humhub-modules-updater/blob/master/views/update/index_noupdate.php#L21

The version that was in the marketplace that we updated to didn't have this button :thinking:

**Comment by luke-** [comment]
Yes, it's not released yet

**Comment by buddh4** [comment]
@ArchBlood also just to make sure, please never add such a menu item (not directly related to your module) in any of your marketplace modules. If you've added this in a custom theme or any non marketplace module its fine.

**Pull request merged**